### PR TITLE
Rename UseExporter to AddExporter

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here. Update the
`CHANGELOG.md` for non-trivial changes.

For significant contributions please make sure you have completed the following items:

* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: examples/AspNet/Global.asax.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ protected void Application_Start()
             this.openTelemetry = Sdk.CreateTracerProviderBuilder()
                  .AddHttpClientInstrumentation()
                  .AddAspNetInstrumentation()
-                 .UseJaegerExporter(c =>
+                 .AddJaegerExporter(c =>
```
Comment (reyang) [inline examples/AspNet/Global.asax.cs:38]: One minor suggestion, it seems we've been very inconsistent in the variable name, for example we use `c` here, and in other places we use `o`, `opt`, `options` - I would vote for changing them all to `options`.

BTW, `this.openTelmetry` is another thing we probably want to change - it should be called `tracerProvider`.

Not blocking this PR though.

File: src/OpenTelemetry.Exporter.Console/ConsoleExporterHelperExtensions.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ namespace OpenTelemetry.Trace
     public static class ConsoleExporterHelperExtensions
     {
         /// <summary>
-        /// Registers a ConsoleActivity exporter to a processing pipeline.
+        /// Adds Console exporter to the TracerProvider.
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Console/ConsoleExporterHelperExtensions.cs:25]: Thanks!

File: src/OpenTelemetry.Exporter.Jaeger/CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@
   were sent as `CHILD_OF`.
   ([#970](https://github.com/open-telemetry/opentelemetry-dotnet/pull/970))
 
+* Rename extension method `UseJaegerExporter` to `AddJaegerExporter`.
```
Comment (reyang) [inline src/OpenTelemetry.Exporter.Jaeger/CHANGELOG.md:9]: I think we need to adding the missing `0.4.0-beta.2` to the log.
I would vote for using passive voice for consistency.

File: src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,8 @@
 * Allow configurable gRPC channel options
   ([#1033](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1033))
 
+* Rename extension method `AddOtlpExporter` to `AddOtlpExporter`.
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md:8]: Looks like a copy/paste fail, should be `UseOtlpExporter` to `AddOtlpExporter` right?

File: src/OpenTelemetry.Exporter.ZPages/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+Rename extension method `AddZPagesExporter` to `AddZPagesExporter`.
```
Comment (CodeBlanch) [inline src/OpenTelemetry.Exporter.ZPages/CHANGELOG.md:5]: Here too, `UseZPagesExporter` to `AddZPagesExporter`.

**Comment by cijothomas** [comment]
All comments addressed. Re-review please.

**Pull request merged**